src/clojure_mauth_client/header.clj

```diff
@@ -1,10 +1,15 @@
 (ns clojure-mauth-client.header
   (:require [pem-reader.core :as pem]
             [clojure.data.codec.base64 :as base64]
-            [clojure.string :refer [blank?]])
+            [jdk.security.Signature :as signature :refer [init-sign update sign]]
+            [clojure.string :refer [blank?]]
+            [clojure.java.io :as io]
+            [jdk.net.URLEncoder :as url-encoder])
   (:use digest)
-  (:import (java.io ByteArrayInputStream)
-           (javax.crypto Cipher KeyGenerator SecretKey)))
+    (:import (java.io ByteArrayInputStream)
+      (javax.crypto Cipher KeyGenerator SecretKey)
+      (java.security Security)
```

Comment:
why are we introducing bouncy castle?

Reply:
I have mimicked algorithm from java mauth library for version 2 to get the private key  - https://github.com/mdsol/mauth-jvm-clients/blob/master/modules/mauth-common/src/main/java/com/mdsol/mauth/util/MAuthSignatureHelper.java#L13

Reply:
Its a heavy library. Can we use an alternative?

Reply:
@priyankaMedidata  and @franklin-mdol  - I have removed bouncy castle dependancy.